trunk/SUAVE/Components/Energy/Networks/Lift_Cruise.py

```diff
@@ -6,6 +6,7 @@
 #           Apr 2021, M. Clarke
 #           Jul 2021, E. Botero
 #           Jul 2021, R. Erhard
+#           Aug 2021, M. Clarke
 
 # ----------------------------------------------------------------------
```

Comment:
In general, the changes made between this script and the other networks are quite similar. Let's update to reduce repeated code

Reply:
Reduced into callable functions